### ISSUE TITLE
Logging into Wiki.js - [networkerror when attempting to fetch resource]

### ISSUE THREAD

**Issue by jkhubner** [Requarks/wiki#2049]
**Question**
Describe in details what you are trying to achieve and where do you need guidance.

Disclaimer: I am fairly new to this sort of thing, but I'm the closest we have to an IT guy.

My end goal is to create a wiki that can be accessed through the internet and edited by multiple members of our company. 

I have created my localhost postgres server, created a database for my wiki, and set it up with no glaring errors, but when I go to log into the wiki for the first time I get the error in the title, though I stay on the login screen. The server and database are active. I don't even know what this means so I'm lost on how to handle it. One thought I had was that I set the port up as 3000 rather than 80, even though I am not behind a reverse-proxy. I did this because my port 80 was in use and changing it seemed like the easier option at the time. 

The url set for the wiki is hosted on dreamhost and should be available for wiki.js, though I don't necessarily know how to confirm that.

Any advice would be much appreciated! Thank you.

**Host Info (please complete the following information):**
OS: Ubuntu 18.04
Wiki.js version: [2.4 ]
Database engine: [postgres 12.3]

**Comment by Smankusors** [comment]
btw, what do you mean "latest" for both Wiki.js and Postgres in your host info?

**Comment by jkhubner** [comment]
I was trying to say they are the current versions. I also missed adding the Postgres which is 12.3.

**Comment by Smankusors** [comment]
you can edit your post if you want 🙂 

anyway, can you read Wiki.js logs when you attempting to login? I guess.... you are not using Docker?

**Comment by jkhubner** [comment]
Where would I find these logs? I am not using Docker. I know about it but have no experience using it.

**Comment by Smankusors** [comment]
I gotta say, using Docker simplifies your installation (you don't need to manually install postgres, node, apache, nginx, etc)

I assume you run it as service, you can use `journalctl -u wiki` as mentioned [here](https://docs.requarks.io/install/linux#run-as-service)

**Comment by jkhubner** [comment]
I believe you, but I suppose since I'm here now it's too late to worry about. 

Well I realized that I had not actually started the service, so thank you for that. However, when I completed the steps. My wiki.js page (localhost:3000) is now unable to connect. A step backwards I suppose.

I have the logs, Not sure if posting them would release sensitive info. But there are a number of failures, the big repeating one being [wiki.service: Failed with result 'exit-code']. Let me know if you'd like to see more.
